style.css

```diff
@@ -0,0 +1,463 @@
+  display: flex;
+  height: 36px;
+  width: 540px;
+  background: white;
+  border-radius: 2px;
+}
+.header-form-search input {
+  border: 0;
+  outline: none;
+  padding: 0 16px;
+  font-size: 14px;
+  width: calc(100% - 32px);
+  justify-content: center;
+  align-items: center;
+}
+.search-icon {
+  color: #2874F1;
+  margin: auto;
+  cursor: pointer;
+}
+/* right pannel */
+.header-rightpan {
+  display: flex;
+  width: 40%;
+  height: 56px;
+  justify-content: flex-start;
+}
+/* login feature */
+.header-login {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  margin: 0 0 0 95px;
+}
+.login-button {
+  color: #2874f0;
+  font-weight: 500;
+  background-color: #fff;
+  cursor: pointer;
+  border-radius: 2px;
+  height: 32px;
+  padding: 5px 40px;
+  border: 1px solid #dbdbdb;
+}
+/* more feature */
+.header-more{
+  justify-content: center;
+  display: flex;
+}
+.more{
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  margin-left: 30px;
+  color: white;
+  font-weight: 500;
+  font-size: 16px;
+  background: none;
+  outline: none;
+  border: none;
+}
+.more-content{
+  display: none;
+  position: absolute;
+  background-color: white;
+  min-width: 230px;
+  box-shadow: 0px 8px 16px 0px rgba(0,0,0,0.2);
+  z-index: 1;
+  margin-top: 56px;
+  font-size: 13px;
+}
+.more-content a{
+  color: black;
+  padding: 12px 16px;
+  text-decoration: none;
+  display: flex;
+  border-bottom: 1px solid #e8e8e8;
+  align-items: center;
+}
+.option-name{
+  margin-left: 10px;
+}
+.more-content a:hover {
+  background-color: #ddd;
+}
+.header-more:hover .more-content {
+  display: block;
+}
+.material-symbols-outlined {
+  font-variation-settings:
+  'FILL' 1,
+  'wght' 500,
+  'GRAD' -25,
+  'opsz' 10
+}
+.more-content .material-symbols-outlined{
+color: #2874f0;
+font-size: 20px;
+fill: #2874f0;
+}
+/* cart feature */
+.header-cart {
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  margin: 0 20px;
+  color: white;
+  font-weight: 500;
+  font-size: 16px;
+}
+/* Category pannel */
+.category-pannel {
+  width: 100%;
+  background: white;
+  justify-content: center;
+  align-items: center;
+  display: flex;
+  min-height: 122px;
+  box-shadow: 0 1px 1px 0 rgb(0 0 0 / 16%);
+}
+.category-items {
+  width: 1280px;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 10px;
+}
+.item {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  gap: 4px;
+  font-size: 14px;
+  font-weight: 500;
+  cursor: pointer;
+}
+.category-items img {
+  height: 64px;
+  width: 64px;
+}
+.item4-name,.item5-name,.item6-name,.item7-name {
+  display: flex;
+  align-items: center;
+}
+.products-container {
+  width: 90%;
+  margin: 10px auto;
+  border-top: 1px solid #eaeaea;
+  border-radius: 2px;
+}
+.deal-ad-container {
+  height: 350px;
+  display: flex;
+}
+.deal-ad-container-2{
+  height: 350px;
+  display: flex;
```

Comment:
If two `div` shares the same css just reuse the class, no need to create a separate class.